trace-utils/src/no_alloc_string.rs

```diff
@@ -0,0 +1,149 @@
+use serde::ser::{Serialize, Serializer};
+use std::borrow::Borrow;
+use tinybytes;
+
+pub struct BufferWrapper {
+    buffer: tinybytes::Bytes,
+}
+
+impl BufferWrapper {
```

Comment:
I'm not sure I understand the advantages of BufferWrapper compared to calling directly the NoAllocString methods with a buffer.